<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtShopview</class>
 <widget class="QWidget" name="QtShopview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="playAnimals1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playAnimals2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playAnimals3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playAnimals4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playAnimals5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals4">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelCoin">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/BoxWhiteBorderBlack.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelheart">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/BoxWhiteBorderBlack.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelRound">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/BoxWhiteBorderBlack #108375.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrize">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/BoxWhiteBorderBlack #108375.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/FieldBuild.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="petBookButton">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../photo/ButtonFlatOrange.png</normaloff>../photo/ButtonFlatOrange.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../photo/ButtonFlatOrange.png</normaloff>../photo/ButtonFlatOrange.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>230</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Sign2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopAnimals5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerLife">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Heart.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerRound">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Hourglass_2x.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerPrize">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Trophy.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerCoin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Coin--Front.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Sign1.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerSpace2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerSpace3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerSpace4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerSpace5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="petBookIcon">
   <property name="geometry">
    <rect>
     <x>1275</x>
     <y>15</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Paw.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="settingsIcon">
   <property name="geometry">
    <rect>
     <x>1335</x>
     <y>15</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Bars.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="playerSpace1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelFright">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>670</y>
     <width>400</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/longButton_fright.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelRefresh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>300</width>
     <height>120</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/longButton_refresh.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelFont1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>255</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>战斗</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelFont2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>商店</string>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace7">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="shopSpace8">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>520</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../photo/Space.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="shopFood1">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shopFood2">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>490</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    border: none;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>playAnimals1</zorder>
  <zorder>playAnimals2</zorder>
  <zorder>playAnimals3</zorder>
  <zorder>playAnimals4</zorder>
  <zorder>playAnimals5</zorder>
  <zorder>shopAnimals1</zorder>
  <zorder>shopAnimals2</zorder>
  <zorder>shopAnimals3</zorder>
  <zorder>shopAnimals4</zorder>
  <zorder>labelCoin</zorder>
  <zorder>labelheart</zorder>
  <zorder>labelRound</zorder>
  <zorder>labelPrize</zorder>
  <zorder>petBookButton</zorder>
  <zorder>settingsButton</zorder>
  <zorder>backgroundLabel1</zorder>
  <zorder>shopAnimals6</zorder>
  <zorder>shopAnimals5</zorder>
  <zorder>playerLife</zorder>
  <zorder>playerRound</zorder>
  <zorder>playerPrize</zorder>
  <zorder>playerCoin</zorder>
  <zorder>backgroundLabel2</zorder>
  <zorder>playerSpace2</zorder>
  <zorder>playerSpace3</zorder>
  <zorder>playerSpace4</zorder>
  <zorder>playerSpace5</zorder>
  <zorder>shopSpace1</zorder>
  <zorder>shopSpace2</zorder>
  <zorder>shopSpace3</zorder>
  <zorder>shopSpace4</zorder>
  <zorder>shopSpace5</zorder>
  <zorder>shopSpace6</zorder>
  <zorder>petBookIcon</zorder>
  <zorder>settingsIcon</zorder>
  <zorder>playerSpace1</zorder>
  <zorder>labelFright</zorder>
  <zorder>labelRefresh</zorder>
  <zorder>labelFont1</zorder>
  <zorder>labelFont2</zorder>
  <zorder>shopSpace7</zorder>
  <zorder>shopSpace8</zorder>
  <zorder>shopFood1</zorder>
  <zorder>shopFood2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
